src/components/MaTabs/MaTabs.vue

```diff
@@ -32,24 +33,32 @@ export default {
         (t) => t.componentOptions.propsData.name === this.selectedTabName
       )
     },
-    setSelectedTabName(name) {
+    setSelectedTabName(name, e) {
+      if (e) e.preventDefault()
       this.selectedTabName = name
       this.$emit('input', name)
     },
+    setHoverTabName(name) {
```

Comment:
Just for consistency with the var name maybe?

```suggestion
    setHoveredTabName(name) {
```